pvanalytics/tests/features/test_orientation.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+from pandas.util.testing import assert_series_equal
+import pandas as pd
+from pvlib import location, pvsystem, tracking, modelchain
+from pvlib.temperature import TEMPERATURE_MODEL_PARAMETERS
+from pvanalytics.features import orientation
+
+
+@pytest.fixture(scope='module')
+def times():
+    """Three days with one hour timestamp spacing in Etc/GMT+7"""
+    return pd.date_range(
+        start='03/01/2020',
+        end='03/04/2020',
+        closed='left',
+        freq='H',
+        tz='Etc/GMT+7'
+    )
+
+
+@pytest.fixture(scope='module')
+def albuquerque():
+    """pvlib Location for Albuquerque, NM."""
+    return location.Location(
+        35.0844,
+        -106.6504,
+        name='Albuquerque',
+        altitude=1500,
+        tx='Etc/GMT+7'
+    )
+
+
+@pytest.fixture(scope='module')
+def system_parameters():
+    """System parameters for generating simulated power data."""
+    sandia_modules = pvsystem.retrieve_sam('SandiaMod')
+    sapm_inverters = pvsystem.retrieve_sam('cecinverter')
+    module = sandia_modules['Canadian_Solar_CS5P_220M___2009_']
+    inverter = sapm_inverters['ABB__MICRO_0_25_I_OUTD_US_208__208V_']
+    temperature_model_parameters = (
+        TEMPERATURE_MODEL_PARAMETERS['sapm']['open_rack_glass_glass']
+    )
+    return {
+        'module_parameters': module,
+        'inverter_parameters': inverter,
+        'temperature_model_parameters': temperature_model_parameters
+    }
+
+
+@pytest.fixture(scope='module')
+def clearsky(times, albuquerque):
+    """Clearsky at `times` in `albuquerque`."""
+    return albuquerque.get_clearsky(times)
```

Comment:
```suggestion
    return albuquerque.get_clearsky(times, model='haurwitz')
```
would avoid needing to import `tables`, if this calculation is why `tables` is needed

Reply:
Looks like 'haurwitz' won't actually work for these tests. `ModelChain.run_model` expects weather to have 'ghi', 'dhi', and 'dni'. When `model='haurwitz'`, `pvlib.Location.get_clearsky()` only returns 'ghi'.

Does`method= 'simplified_solis'` require tables?

Reply:
> Doesmodel='simplified_solis' require tables?

It doesn't, and would serve the purpose here.